project.json

```diff
@@ -22,19 +23,33 @@
         },
         "Microsoft.Extensions.Configuration.FileProviderExtensions": "1.0.0-*",
         "Octokit": "0.19.0",
-        "Microsoft.Net.Http": "2.2.29"
+        "Microsoft.Net.Http": "2.2.29",
+        "Microsoft.EntityFrameworkCore.Commands": {
+            "version": "1.0.0-*",
+            "type": "build"
+        }
     },
 
     "frameworks": {
         "netcoreapp1.0": {
             "imports": [
+                "dnxcore50",
+                "dotnet5.4",
                 "netcore45",
-                "dotnet5.4"
+                "portable-net45+win8"
             ]
         }
     },
 
     "tools": {
+        "dotnet-aspnet-codegenerator": "1.0.0-*",
+        "dotnet-ef": {
+            "version": "1.0.0-*",
+            "imports": [
+                "dnxcore50",
+                 "portable-net45+win8"
```

Comment:
> It seems EF isn't aware of netcoreapp yet?

Correct. Unless you're using a custom build, current versions of EF don't work on netcoreapp1.0 yet. Try changing your tfm to netstandard1.5 for now as a workaround.